README.md

```diff
@@ -30,18 +30,29 @@ Start a local authentik instance by following https://goauthentik.io/docs/instal
 Before starting the instance, add this static token to the `.env` file:
 
 ```
-AUTHENTIK_TOKEN=this-password-is-for-testing-dont-use
+AUTHENTIK_TOKEN=this-token-is-for-testing-dont-use
 ```
 
 Afterwards, tests can be run from your Editor or via CLI:
 
 ```
 export TF_ACC=1
 export AUTHENTIK_URL=http://localhost:9000
-export AUTHENTIK_TOKEN=this-password-is-for-testing-dont-use
-go test -timeout 30s ./... -count=1
+export AUTHENTIK_TOKEN=this-token-is-for-testing-dont-use
+go test -timeout 0 ./... -count=1
```

Comment:
I'd rather increase the timeout that set it to 0 and have that process hang indefinitely on a contributor's machine.

Reply:
The tests seem very CPU-intensive on the local Authentik server, so I'm not sure what a safe but sane timeout might be when it can take 2 minutes or so on my machine and 8 minutes in the GitHub pipeline.
I suppose we could set it to like 30 minutes? lol